diagnostic.md

```diff
@@ -150,7 +176,9 @@ div {
 ```
 
 <!-- your answer starts here -->
+all divs: left side of the page except for...
 
+divOne will be on the right side.
 <!-- your answer ends here -->
```

Comment:
.divOne will be in the top-right corner; .divTwo will be in the top-left corner, with .divThree immediately to its right; .divFour will be underneath .divTwo, and .divFive will sit immediately to the right of .divFour (under .divThree).

In short: 2 3 1 4 5